agent/php_nrini.c

```diff
@@ -2206,6 +2206,22 @@ STD_PHP_INI_ENTRY_EX("newrelic.framework.wordpress.hooks",
                      zend_newrelic_globals,
                      newrelic_globals,
                      nr_on_off_dh)
+STD_PHP_INI_ENTRY_EX("newrelic.framework.wordpress.hooks_threshold",
+                     "1ms",
+                     NR_PHP_REQUEST,
+                     nr_time_mh,
```

Comment:
If an invalid valid is given I think `nr_time_mh` will return `0`.  This will set the threshold to `0` and lots of metric will be generated.

Reply:
@mfulb Good catch! However, `nr_time_mh` is already used in other places where 0 will cause additional data to be generated by the agent. See [here](https://github.com/newrelic/newrelic-php-agent/blob/533f7bd0f4c4775c090d9754072a194fdb0b5ddb/agent/php_nrini.c#L2430-L2439), [here](https://github.com/newrelic/newrelic-php-agent/blob/533f7bd0f4c4775c090d9754072a194fdb0b5ddb/agent/php_nrini.c#L2440-L2449). What would you suggest be done about it?

Reply:
For this particular one I would create a `nr_time_wordpress_hooks_mh` that enforces the default if an invalid value is given.  Perhaps it can be made to leverage the current `nr_time_mm` for the value parsing.

Reply:
Also see #802, which will change the default value for `newrelic.framework.wordpress.plugins` from `false` to `true` (and such settings obsoletes `newrelic.framework.wordpress.hooks_threshold`) as explained by here: https://github.com/newrelic/newrelic-php-agent/pull/802#issuecomment-1856585843.

Reply:
This appears to be an issue that needs a more comprehensive handling since it affects several INI values.  For this PR the existing behavior seems adequate.